docs/GettingStarted.md

```diff
@@ -595,3 +595,4 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- NTOS eBPF extensions: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
```

Comment:
There are two separate extensions there that I'd prefer be listed separately here with a link to their own readme file.
ntosebpfext however has no README file and so isn't really documented (this is https://github.com/microsoft/ntosebpfext/issues/95).
And neteventebpfext is more about network events than "NTOS" per se, so calling it an NTOS extension is a misnomer in my view.  If I were looking for network events, I wouldn't look under "NTOS" extensions, and so wouldn't find it.

Reply:
Thanks for the suggestion. Until the readme is committed, I have pointed ntosebpfext to the main page.